README.md

```diff
@@ -15,35 +15,53 @@ to normalize the strings in your database.
 
 ## Install
 
-Just `npm install --save prx-podagent`. It's that easy!
+Just `npm install --save prx-podagent`. Or to use outside of node, just grab a
+database file out of the `db/` directory.
 
 ## Usage
 
-Async (recommended):
-
 ```node
 const podagent = require('prx-podagent');
-podagent.parse('some-string', (err, agent) => {
-  if (agent) {
-    console.log('Match:', agent.name, agent.type, agent.os);
-  } else {
-    console.log('Did not match any known agents');
-  }
-});
-```
-
-Sync:
-
-```node
-const podagent = require('prx-podagent');
-let agent = podagent.parse('some-string');
+const agent = podagent.parse('some-string');
 if (agent) {
   console.log('Match:', agent.name, agent.type, agent.os);
 } else {
   console.log('Did not match any known agents');
 }
 ```
 
+Or to DIY use the json non-lock file:
+
+```javascript
+  $.getJSON('agents.json', function(db) {
+    var str = 'some-agent-string';
+    var agent = db.agents.find(function(a) { return eval(a.regex).test(str); });
```

Comment:
I'm now wondering if this was a bad idea ... most languages don't have a "safe" way to turn a `/regex/i` into a RegExp object.  Without some sort of eval.  Vs storing the regex string and flags separately.